generators/entity-server/templates/src/test/java/package/web/rest/EntityResourceIT.java.ejs

```diff
@@ -170,9 +170,15 @@ import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.
 <%_ } else { _%>
 @SpringBootTest(classes = <%= mainClass %>.class)
 <%_ } _%>
-<%_ if (cacheProvider === 'redis') { _%>
-@ExtendWith(RedisTestContainerExtension.class)
+<%_ if (searchEngine === 'elasticsearch' || fieldsContainOwnerManyToMany === true || cacheProvider === 'redis') { _%>
+@ExtendWith(<% if (cacheProvider === 'redis') { %>{ RedisTestContainerExtension.class, MockitoExtension.class }%><% } else if (searchEngine === 'elasticsearch' || fieldsContainOwnerManyToMany === true) { %>MockitoExtension.class<%}%>)
+<%_ } _%>
```

Comment:
I think it might be simpler that way:
```
<%_ if (cacheProvider === 'redis') { _%>
@ExtendWith({ RedisTestContainerExtension.class, MockitoExtension.class })
<%_ } else if (searchEngine === 'elasticsearch' || fieldsContainOwnerManyToMany === true) { %>
@ExtendWith(MockitoExtension.class)
<%_ } _%>
```